defaults/main.yml

```diff
@@ -28,3 +28,11 @@ nms_clickhouse_release_kind: lts
 
 ## By default, NMS certificates are removed by default
 nms_remove_certs: true
+
+## By default, no NMS modules are installed.
+## If version is not set, the latest version is installed.
+## If setup is not set, the value from the nms_setup is used.
+nms_modules: []
+  # - name: nms-api-connectivity-manager
```

Comment:
Guessing this is an example of what one would do. Maybe update this to `acm` now that the input have changed?

Reply:
Thanks @jswongF5, this has been address in the latest changes